src/main/java/net/tropicraft/core/common/data/TropicraftItemTagsProvider.java

```diff
@@ -86,29 +87,29 @@ protected void registerTags() {
         appendToTag(ItemTags.MUSIC_DISCS, MUSIC_DISCS);
 
         // Copy block tags
-        copy(TropicraftTags.Blocks.SAND, SAND);
+        copyBlockAndAppendToTag(TropicraftTags.Blocks.SAND, SAND, ItemTags.SAND); // IDK know about that
```

Comment:
I do not know if this is wanted since I read some other pull requests were people tried to add the ability to convert pruified sand to normal sand.

Reply:
We don't want our sand to be included in the Vanilla tag, could this be removed?

Reply:
Actually hold off for a bit on this, we're having discussions about it